modules/internal/localeModels/flat/LocaleModel.chpl

```diff
@@ -155,6 +155,7 @@ module LocaleModel {
       nPUsLogAcc = 0;
       nPUsLogAll = 0;
       maxTaskPar = 0;
+      numColocales = 0;
```

Comment:
I suspect the corresponding change is needed in `localModels/gpu/LocaleModel.chpl`

Reply:
Done. Thanks for catching that.